sourced/ml/cmd_entries/args.py

```diff
@@ -29,37 +30,37 @@ def add_vocabulary_size_arg(my_parser: argparse.ArgumentParser):
         help="The maximum vocabulary size.")
 
 
-def add_extractor_args(my_parser: argparse.ArgumentParser):
+def add_repo2_args(my_parser: argparse.ArgumentParser, default_packages=None):
     my_parser.add_argument(
         "-r", "--repositories", required=True,
         help="The path to the repositories.")
     my_parser.add_argument(
-        "-l", "--languages", required=True, nargs="+", choices=(
-            # TODO(zurk): get languages from bblfsh directly as soon as
-            # https://github.com/bblfsh/client-scala/issues/68 resolved
-            "Java", "Python", "Go", "JavaScript", "TypeScript", "Ruby", "Bash", "Php"),
+        "--parquet", action="store_true", help="Use Parquet files as input.")
+    my_parser.add_argument(
+        "--graph", help="Write the tree in Graphviz format to this file.")
+    my_parser.add_argument(
+        "-l", "--languages", required=True, nargs="+",
+        choices=("Java", "Python", "Go", "JavaScript", "TypeScript", "Ruby", "Bash", "Php"),
+        default=["Java", "Python", "Go", "JavaScript", "TypeScript", "Ruby", "Bash", "Php"],
```

Comment:
Konst's TODO still holds. Also please use the same variable twice to avoid duplication

Reply:
my bad, didnt mean to delete it, on it

Reply:
also gonna take out the required since it will default to all languages now

Reply:
my bad, didnt mean to delete it, on it. Im also gonna delte the required=True since we default to all languages now